framework/src/play/src/main/scala/play/api/mvc/RequestHeader.scala

```diff
@@ -265,7 +267,7 @@ trait RequestHeader {
    * @return A new request with the body attached to the header.
    */
   def withBody[A](body: A): Request[A] =
-    new RequestImpl[A](connection, method, target, version, headers, attrs, body)
+    new RequestImpl[A](connection, method, target, version, headers, attrs, tags, body)
```

Comment:
Tags are something we're trying to remove from Play in this release. If it's slow to access a tag then we should move the usage to an attribute instead.